README.Rmd

```diff
@@ -98,6 +98,10 @@ For R Shiny modules, use the following naming pattern:
 Any functions from `shiny` and `shinyFiles` can be used directly without the `shiny::` prefix.
 For any other packages, be explicit and use `mypackage::myfunction()`.
 
+#### Before Making a Release

```

Comment:
this refers to a release of the HIA program, not the hiaR package, right? -> I would make this specific about for which software a release is made.